README.md

```diff
@@ -171,4 +166,4 @@ reload, and you should now be using obfuscation.
    * `npm install -g wup`
  * Run wup: `(cd data ; wup) &`
  * Then, run the benchmark with `npm run benchmark`.
- 
+ 
```

Comment:
Good eyes but no! There were two blank lines at the end of the file. I don't know why.